tests/test_plot_time_series.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pandas as pd
+import numpy as np
+from plot_time_series import plot_time_series
+
+def create_test_data():
```

Comment:
Let's follow the convention using `@pytest.fixture`. You can refer to the other testing files.